src/Behat/Gherkin/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -23,7 +23,7 @@ public function supports($resource)
 
     public function load($resource)
     {
-        return array_values(array_map(
+        return array_filter(array_map(
```

Comment:
I would actually still keep the `array_values` call so that we return a list rather than a array with holes.